addons/mail/static/src/components/activity/activity.js

```diff
@@ -128,8 +128,9 @@ class Activity extends Component {
      * @param {Object} ev.detail
      * @param {mail.attachment} ev.detail.attachment
      */
-    _onAttachmentCreated(ev) {
-        this.activity.markAsDone({ attachments: [ev.detail.attachment] });
+    async _onAttachmentCreated(ev) {
+        await this.activity.markAsDone({ attachments: [ev.detail.attachment] });
+        this.trigger('o-attachments-changed');
```

Comment:
Reminder: adapt in 15.2 to use `reloadParentView`.
It's also the actual version that we stop using `t-on` bubbling in discuss, as it's no longer supported in 15.3